### ISSUE TITLE
Button to refresh script connection (to delete temporary tables)

### ISSUE THREAD

**Issue by ishazel** [microsoft/azuredatastudio#20768]
Hi, huge fan of ADS 👏🏻

While working on reports, I often end up disconnecting then reconnecting to the server in order to quickly delete all temporary tables so I can hit the "run" button without issue to test the script. It would be nice if there was a button to refresh an existing connection.

**Comment by cheenamalhotra** [comment]
Hi @ishazel 

ADS does support 'Refresh' on existing connections, may I know which version of ADS you're using?

![image](https://user-images.githubusercontent.com/13396919/195142212-34122fe1-87c5-4322-9905-d9fda8dc7fd2.png)
_(Captured from Azure Data Studio v1.39.1)_

**Comment by ishazel** [comment]
Ah I didn't think to look there, I was thinking of a refresh button in the toolbar and/or a command.

That said, using the method you shared does not achieve the same result as disconnecting then reconnecting the query to its connection. Refreshing from the connections panel does not seem to clear temporary tables.

I'm using Insiders 1.40.0

**Comment by cheenamalhotra** [comment]
Yes, that is not going to clear temp tables created in query window as in both places, physical connections to database are different. The possible way right now is to disconnect and reconnect as you mentioned. 

We'll look into this!

**Comment by ishazel** [comment]
Thank you! I hope one day I can help contribute to this amazing piece of software 🥰
